_posts/2020-01-30-Age-Targeting-Part1.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Increase your target audience via ML on Spark [part 1]
+subtitle: Ground truth selection
+category: data
+tags: [adtech, howto, spark]
+author: Alessio Borgheresi
+header-img: "images/Age-Targeting/true_age.jpg"
+---
+
+
+In mobile advertising it is possible to target the audience of the campaign based on some charateristics of the users.
+Here we are focusing on the age.
+However, these informations are only provided on by a subset of apps during the bidding process.
+To increase the available targeting audience we would need an estimation of the age for the devices which never use those apps and here is when the machine learning comes into place.
+
+To train a supervised machine learning for infearing the age over all the devices we need: 
+- a **reliable ground truth** datasets (in our case the age for certain devices);
+- the **features** which will go as inputs to the model.
+
+The ground truth has to be reliable or the model will learn non realistic structure.
+Moreover, it could improve the performances of the training.
+In this post it is discussed only the ground truth selection step, leaving the feature creation, the tuning, the train and the prediction of the model for the next time.
+
+Reliability is not a statistic quantity that you can find in the books.
+To summarize the concept in one phrase, we could state that your data are reliable as long as all your results follow an explainable behaviour without any inconsistency results.
+It is based on this idea that we have studied and discovered some pattern in the data that in the end we prefer to exclude from the ground truth dataset.
```

Comment:
General rule: The english language tends to prefer simple sentences.

This is based on the fact, that we often find patterns which imply some problems in the data. We want to discover all of those patterns and exclude them from our dataset.